source/core/parameters.cc

```diff
@@ -337,6 +337,29 @@ namespace Parameters
         "Enable calculation of total enstrophy. The total enstrophy "
         "is calculated from the volumetric integral of the enstrophy over the domain.");
 
+      prm.declare_entry(
+        "calculate average velocities",
+        "false",
+        Patterns::Bool(),
+        "Enable calculation of average velocity and reynolds stress profiles.");
+
+      prm.declare_entry(
+        "initial time",
+        "0.0",
+        Patterns::Double(),
+        "Initial time to start calculations for average velocities");
+
+      prm.declare_entry("boundary id",
+                        "0",
+                        Patterns::Integer(),
+                        "Boundary id of the inlet flow");
+
+      prm.declare_entry("flow direction",
+                        "0",
+                        Patterns::Integer(),
+                        "Flow direction at flow inlet");
```

Comment:
I do not understand why those are needed for average velocities. I really don't think it's worth it to add all these parameters when all they will do is allow you to divide by a constant which could easily be done in paraview or elsewhere.

Reply:
I thougth I removed it. this is a mistake, those parameters are deleted in the .h.